src/Collections/Definitions.php

```diff
@@ -39,6 +39,10 @@ protected function doExport()
 
     /**
      * Returns the schema for the given field.
+     *
+     * @param string $name
+     *
+     * @return Schema
```

Comment:
I would not add php docs not bringing new information, it just makes the code harder to read imo.

Reply:
True, I will remove it